src/interface/meshblock_data.cpp

```diff
@@ -815,6 +816,50 @@ TaskStatus MeshBlockData<T>::ClearBoundary(BoundaryCommSubset phase) {
   return TaskStatus::complete;
 }
 
+template <typename T>
+void MeshBlockData<T>::RestrictBoundaries() {
+  auto pmb = GetBlockPointer();
+  pmb->pbval->RestrictBoundaries();
+  // TODO(JMM): once we have variable-level pbvals we can do this
+  /*
```

Comment:
is this dead code or did you intend to keep it around?

Reply:
This code never worked. Its essentially an aspirational comment for how I'd like things to look if we refactor the `BoundaryValues` class.

Reply:
I'm going to delete this code as it never worked. But I will leave the TODO in.